modules/doc/training/content/sqa/training/lead/sqa_release.md

```diff
@@ -62,17 +62,17 @@ The tag.gz files should be placed in the "release" directory.
 
 ## Perform Release Review
 
-PLN-4005 details the necessary steps for performing the release which includes performing a
-final review of MOOSE or MOOSE-based application. This review +must+ be performed by the Project Lead
-and is intended to ensure that all requirements are satisfied and that all [!ac](SQA) documentation
-is correct and complete.
+PLN-4005 details the necessary steps for performing the release which includes performing a final
+review of a MOOSE or MOOSE-based application. This review +must+ be performed by the Project Lead and
```

Comment:
```suggestion
review of MOOSE or a MOOSE-based application. This review +must+ be performed by the Project Lead and
```

Oops, somehow my brain flipped this around to the "right" way when I reviewed the first time.